src/sentry/south_migrations/0498_backfill_installation_status.py

```diff
@@ -0,0 +1,1484 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+from sentry.constants import SentryAppInstallationStatus
+
+
+class Migration(DataMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+    def _forwards(self, orm):
+        "Write your forwards methods here."
+        # Note: Don't use "from appname.models import ModelName".
+        # Use orm.ModelName to refer to models in this application,
+        # and orm['appname.ModelName'] for models in other applications.
+        SentryAppInstallation = orm['sentry.SentryAppInstallation']
+
+        installs = SentryAppInstallation.objects.filter(
+            api_token_id__isnull=False,
+            date_deleted__isnull=True,
+        ).update(status=SentryAppInstallationStatus.INSTALLED)
```

Comment:
@MeredithAnya Will `update` trigger the `save` function of that model? If not, we should manually update `date_updated` as well.